MsCorePkg/MacAddressEmulationDxe/Test/MacAddressEmulationDxeHostTestCommon.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <stdio.h>
+#include <string.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <cmocka.h>
```

Comment:
I noticed a lot of places the code is using standard C items like these .H files, size_t, int, etc.  Is there a reason we can't use the standard UEFI H files and definitions?  Will this code be compiled outside UEFI?

Reply:
I see in the TestMain.c file below there is a standard POSIX C main() function, so I assume you are compiling 2 ways.  Is there a way we can use just UEFI definitions in that situation so engineers digging into this only have to work with 1 coding standard?

Reply:
Yes, these are host based unit tests designed to compile only in Linux/Windows.

I'm a bit new to UEFI unit test framework, but at HP we had a custom unit test framework and at some level you do need to call the OS functions for the actual implementation.

I did copy these headers from an existing test, possible they aren't needed since the `UnitTestLib` should include them and stub the UEFI functionality internally.

Reply:
Looks like those headers are required for the end-test file to include in order to compile, will leave them in.